DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/Extensions/SecurityContext/SecurityContextUserExtension.cs

```diff
@@ -8,10 +8,14 @@ namespace GitHubExtension.Security.WebApi.Extensions.SecurityContext
 {
     public static class SecurityContextUserExtension
     {
-        public static IEnumerable<User> GetUsersByName(this ISecurityContextQuery securityContextQuery, string userName)
+        public static IEnumerable<User> GetUsersByNameExceptCurrent(
```

Comment:
But I don't see any unit tests on this extension method. Are you sure that it's not braking changes?